api/src/applicationStatus/applicationStatus.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Application } from '@/application/application.entity';
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+
+// import { CustomBaseEntity } from '../common/custom-base.entity';
```

Comment:
can we remove comments?

Reply:
Yep, I'll do it with `COMPLETION_STATUS` changes